### DISCUSSION TITLE
Meeting Guide not Syncing with GSO

### DISCUSSION THREAD

**Discussion by fbarnese** [code4recovery/12-step-meeting-list#1279] [General]
Hi folks.
We have received a notice from the NY tech team that they are not able to sync with our meeting directory.  Nothing has changed on our side.  Could someone take a look and advise?  www.oc-aa.org
Faith

**Comment by joshreisner** [comment]
hi @fbarnese do you have access to the TIAA forum? sites using GoDaddy have experienced this same issue and have shared experiences fixing it here: https://discourse.tiaa-forum.org/t/meeting-guide-app-cannot-access-json-file-error-403/9257/18

**Reply by lovethecodelabs** [reply]
If it’s managed wordpress, turn off cdn in settings and make sure file permissions are correct

Guy said wp- config.php specifically
